ee/tuf/library_lookup.go

```diff
@@ -38,7 +39,7 @@ var channelsUsingNewAutoupdater = map[string]bool{
 // For now, it is only available when launcher is on the nightly update channel.
 func CheckOutLatestWithoutConfig(binary autoupdatableBinary, logger log.Logger) (*BinaryUpdateInfo, error) {
 	logger = log.With(logger, "component", "tuf_library_lookup")
-	cfg, err := getAutoupdateConfig()
+	cfg, err := getAutoupdateConfig(os.Args[1:])
```

Comment:
You think os.Args, or should we be passing it in?

Reply:
Mostly just moved `os.Args[1:]` up from where it previously was in `getAutoupdateConfig()` so that I could unit test it. I don't have a strong opinion.

Reply:
@directionless I'm going to merge this as-is to move this forward, but I'm going to be doing another PR in this same area and I'm can update this there if you'd like